common/postgresql-ng/templates/configmap.yaml

```diff
@@ -34,38 +34,41 @@ data:
       {{- $users := merge $users (dict "metrics" (dict)) }}
     {{- end }}
 
-    -- pw_method is set in the entrypoint where psql executes this sql file
-    SET password_encryption = :'pw_method';
+    {{- if lt ($.Values.postgresVersion | int) 14 }}
+    CREATE ROLE poor_mans_pg_read_all_data;
+    SELECT 'CREATE ROLE "poor_mans_pg_read_all_data"' where not exists (select rolname from pg_authid where rolname = 'poor_mans_pg_read_all_data')\gexec
```

Comment:
The first line appears to be superfluous with the second. Also, I don't see permissions being granted to `poor_mans_pg_read_all_data`.